go.mod

```diff
@@ -132,7 +133,7 @@ require (
 	github.com/sirupsen/logrus v1.9.2
 	github.com/stretchr/testify v1.9.0
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/triplewz/poseidon v0.0.1
+	github.com/triplewz/poseidon v0.0.2-0.20240407130934-5265fab9d889
```

Comment:
Any chance we could ask for an upstream release?

Reply:
Not sure how to get to the maintainer, @vmx might know though.

Reply:
(I wouldn't block this PR on the upstream tag, this is actively blocking us on updating Curio to latest NV, and Curio is now most of calibnet)

Reply:
I've filed an issue, no reason to wait.